### PR TITLE
Create 2022-11-rse-hps-workshop.md

### PR DESCRIPTION
**Pull request by rinkug**
Addresses #1383

### CODE REVIEW

File: Events/2021-11-rse-hps-workshop.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Research Software Engineers in HPC Workshop at SC22 (RSE-HPC-2022)
+- Date: July 11, 2022
```
Comment (cferenba) [inline Events/2021-11-rse-hps-workshop.md:2]: Is this supposed to be today's date, or the submission deadline (August 5)?

**Review by cferenba — CHANGES_REQUESTED** [review]

File: Events/2021-11-rse-hps-workshop.md (hunk 1 above)
Comment (rinkug) [reply Events/2021-11-rse-hps-workshop.md]: Should we submission deadline. I am fixing this.

**Comment by rinkug** [comment]
@cferenba : Does this look fine: https://preview.bssw.io/events/research-software-engineers-in-hpc-workshop-at-sc22-rse-hpc-2022

This is going to be published today evening.

**Comment by cferenba** [comment]
@rinkug I can't view the preview, but the information in the .md file looks fine.

**Comment by rinkug** [comment]
> @rinkug I can't view the preview, but the information in the .md file looks fine.

Ah. You need credentials. I can send you over email if you want to look at it. It looks fine IMO.

**Comment by cferenba** [comment]
If you think it looks fine, that's good enough for me.

**Pull request merged**